common/persistence/domainReplicationQueue.go

```diff
@@ -51,8 +59,10 @@ func NewDomainReplicationQueue(queue Queue, logger log.Logger) DomainReplication
 
 type (
 	domainReplicationQueueImpl struct {
-		logger              log.Logger
 		queue               Queue
+		clusterName         string
```

Comment:
current cluster name?

Reply:
It's called clusterName everywhere else so I'm going to keep it as is.